Bukkit/src/main/java/com/plotsquared/bukkit/listener/BlockEventListener.java

```diff
@@ -277,7 +277,7 @@ public void blockCreate(BlockPlaceEvent event) {
         BukkitPlayer pp = BukkitUtil.adapt(player);
         Plot plot = area.getPlot(location);
         if (plot != null) {
-            if ((location.getY() > area.getMaxBuildHeight() || location.getY() < area
+            if ((location.getY() >= area.getMaxBuildHeight() || location.getY() < area
```

Comment:
`>` because max build height is inclusive?

Reply:
That's a mistake in the javadocs for the method. It's exclusive to maintain compatibility with previous versions (256 used to be the max, not 255)

Reply:
Hmm, though before this PR the check was with a `>`, so that would suggest inclusive. Maybe that was a bug haha. The default value of `maxBuildHeight` in `PlotArea` is currently `PlotSquared.platform().versionMaxHeight()` by the way, which is inclusive.

Reply:
That's been fixed now